src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -67,6 +67,8 @@
       <!-- Add Scoped CSS files in the app to the list of hidden items -->
       <_TemporaryHiddenContent Include="Pages\**\*.razor.css" />
       <_TemporaryHiddenContent Include="Shared\**\*.razor.css" />
+      <!-- Add global.json to the list of hidden items -->
+      <_TemporaryHiddenContent Include="global.json" />
```

Comment:
**Note:** Issue also repros with other `json` files, this isn't unique to `global.json`. Should we be filtering out _all_ `json` files from the bundle resources?

Reply:
> **Note:** Issue also repros with other `json` files, this isn't unique to `global.json`. Should we be filtering out _all_ `json` files from the bundle resources?

Seems like in that case it would be easier to just not include them:

https://github.com/dotnet/sdk/blob/6698cd97f38ea3ad2c7182f95e5b9ec887c29f56/src/RazorSdk/Sdk/Sdk.Razor.StaticAssets.ProjectSystem.props#L33

Reply:
I'd considered that approach as well, however given this issue was iOS specific I wanted to try out a more targeted fix. @javiercn thoughts on this?

Reply:
Not specific to this PR, but rather these exclusions in general: Everything about all of this is hacky, so I have a hard time making any rational statements about this. There's just no way to know what the user truly intended in the general case, so we end up with a _heuristic_, which means "we **know** this is wrong, but it's better than the alternative of having nothing."

So, party on 😁

Reply:
BTW Android does have some similar restrictions, but not quite the same. Just look at all the other workarounds in the targets 😄

Reply:
> @rolfbjarne do we know why MAUI iOS/MacCatalyst doesn't like these files

1. For iOS and Mac Catalyst any Content files are (supposed to be) added the app bundle.
2. That requires knowing where to put the files inside the app bundle.
3. There's a default logic (compute the relative path relative to the project file), and that's where Content files are located inside the app bundle.
4. When applying this logic to the global.json file this issue is about, the resulting file in the app bundle turns out to be way outside the app bundle (you can see the computed path in the error message).
5. This is obviously not correct, so we show an error.

It's possible to override what's done about Content files by setting the `PublishFolderType` metadata (setting `PublishFolderType=None` would make iOS/Mac Catalyst not try to add the file to the app bundle)

Reply:
> It's possible to override what's done about Content files by setting the `PublishFolderType` metadata (setting `PublishFolderType=None` would make iOS/Mac Catalyst not try to add the file to the app bundle)

It might be reasonable if the .NET SDK (or the Razor/web SDK) applied this. I can't see a reason why global.json should be ever published.

```xml
<ItemGroup>
  <Content Update="@(Content)" Condition="'%(Content.FileName)' == 'global' and '%(Content.Extension)' == '.json'" PublishFolderType=None />
</ItemGroup>
```

Should do the trick, I think.

Reply:
https://github.com/dotnet/maui/pull/8724/commits/172cadeb6212fc2e64dfb792b7d25847737083be updates to use `PublishFolderType` within this target. @javiercn would this be appropriate for now, or do you think it's worthwhile updating the SDK with this exclusion?

Reply:
I think this is fine, but it is probably best if it lived in the SDK

Reply:
I'll take a look next time I make SDK related changes. Want to avoid the whole flow just for this fix right now.